docs/src/userguide/parallelism.rst

```diff
@@ -190,4 +190,4 @@ OpenMP functions can be used by cimporting ``openmp``:
 
 .. rubric:: References
 
-.. [#] http://www.openmp.org/mp-documents/spec30.pdf
+.. [#] https://www.openmp.org/wp-content/uploads/spec30.pdf
```

Comment:
The new URL looks like an implementation detail. Better keep the old one (+https).